integrations/optimum/src/haystack_integrations/components/embedders/optimum/optimization.py

```diff
@@ -7,7 +7,7 @@
 
 class OptimumEmbedderOptimizationMode(Enum):
     """
-    [ONXX Optimization Modes](https://huggingface.co/docs/optimum/onnxruntime/usage_guides/optimization.html)
+    [ONXX Optimization modes](https://huggingface.co/docs/optimum/onnxruntime/usage_guides/optimization.html)
```

Comment:
```suggestion
    [ONXX Optimization modes](https://huggingface.co/docs/optimum/onnxruntime/usage_guides/optimization)
```
The original URL does not seem to work.